conftest.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 import matplotlib
+import plotly
```

Comment:
I see that you've added plotly as an extra requirement in the setup.py, but if it's not installed then this line returns an error, for example, with `pytest --unit -v`. Once installed of course, there is no issue. Should plotly be required here?

Reply:
Good catch, the workflow for this is for developers to install the package with the [all] flag (i.e. `pip install -e '.[all]'`) which provides plotly. An update of the contributing.md is required to capture this change.

Reply:
We also recommend that new users run `pytest --unit -v` at the end of the Installation section of the Readme.